doc/md/examples/count-v1.mo

```diff
@@ -1,7 +1,7 @@
 import Debug "mo:base/Debug";
 
-actor Counter_v1 {
-  stable var state : Nat = 0;
+persistent actor Counter_v1 {
+  var state : Nat = 0; // implicitly `stable`
```

Comment:
better search&replace

Reply:
Whaddy mean?

Reply:
Ok, I see what you mean. But the keyword is still stable, not persistent.